frontend/admin/src/js/components/user/GeneralInformationTab.tsx

```diff
@@ -0,0 +1,676 @@
+          })
+          .catch(() => {
+            toast.error(
+              intl
+                .formatMessage({
+                  defaultMessage: "Failed updating notes for candidate in",
+                  description:
+                    "Toast notification for failed update of candidates notes",
+                })
+                .concat(` ${candidate.pool?.name?.[locale]}` || ""),
+            );
+          });
+      }
+    });
+  };
+
+  return (
+    <>
+      <p>
+        {intl.formatMessage({
+          defaultMessage:
+            "These notes are shared between all managers of this pool, but not to candidates.",
+          description:
+            "Message about the behavior of notes on the view-user page",
+        })}
+      </p>
+      {isEmpty(user.poolCandidates) ? (
+        <div
+          data-h2-bg-color="b(lightgray)"
+          data-h2-padding="b(all, s)"
+          data-h2-radius="b(s)"
+          data-h2-font-size="b(caption)"
+        >
+          {intl.formatMessage({
+            defaultMessage: "This user is not in any pools yet",
+            description:
+              "Message on view-user page that the user is not in any pools",
+          })}
+        </div>
+      ) : (
+        <BasicForm onSubmit={handleSubmit}>
+          {user?.poolCandidates?.map((candidate) => {
+            if (candidate) {
+              return (
+                <div data-h2-padding="b(bottom, s)" key={candidate.id}>
+                  <TextArea
+                    id={candidate.id}
+                    name={candidate.id}
+                    label={`${intl.formatMessage({
+                      defaultMessage: "Notes",
+                      description: "Title for a pool candidates notes field",
+                    })} - ${candidate.pool?.name?.[locale]}`}
+                    defaultValue={candidate.notes ? candidate.notes : ""}
+                    placeholder={intl.formatMessage({
+                      defaultMessage: "Start writing your notes here...",
+                      description:
+                        "Placeholder text for a pool candidates notes field",
+                    })}
+                    rows={4}
+                  />
+                </div>
+              );
+            }
+            return null;
+          })}
+          <Button type="submit" mode="solid" color="secondary">
+            <span data-h2-font-style="b(underline)">
+              {intl.formatMessage({
+                defaultMessage: "Save notes",
+                description:
+                  "Button to save notes for a pool candidate on the view-user page",
+              })}
+            </span>
+          </Button>
+        </BasicForm>
+      )}
+    </>
+  );
+};
+
+const EmploymentEquitySection: React.FC<BasicSectionProps> = ({ user }) => {
+  const intl = useIntl();
+
+  return (
+    <div
+      data-h2-bg-color="b(lightgray)"
+      data-h2-padding="b(right-left, s) b(top-bottom, xxs)"
+      data-h2-radius="b(s)"
```

Comment:
Other sections have this. May be a good idea to add it here as well for consistency.

```suggestion
      data-h2-radius="b(s)"
      data-h2-font-size="b(caption)"
```

Reply:
@Jerryescandon Could you please confirm the font sizes for the various sections are as intended when you review the UI?

Reply:
I'll try to bring the heading styles more in line but I was actually wondering about the difference between the text here:
![image](https://user-images.githubusercontent.com/49166751/173446953-fee4c03b-7f2f-4bef-a6cd-7d88a16b6b98.png)
and here:
![image](https://user-images.githubusercontent.com/49166751/173447088-b1f8ed22-6c31-470c-aa85-371522a7dcbb.png)
![image](https://user-images.githubusercontent.com/49166751/173447159-533662fa-ddc1-49b2-8449-84187f2367e6.png)

Reply:
Am I right in thinking those heading styles are a general rule for the whole site (or just admin)? If so it might be good idea to create a new issue for bringing everything more in line with this.

Reply:
So the only thing I would recommend would be to increase these up to the same size as "body" used at the bottom.


![image](https://user-images.githubusercontent.com/16977254/173619294-7fce1582-2b54-486e-89d9-8d8d4d6c7313.png)

Reply:
> Am I right in thinking those heading styles are a general rule for the whole site (or just admin)? If so it might be a good idea to create a new issue for bringing everything more in line with this.

I would think Admin for now, Im unsure where we are headed with the public facing stuff. So I would hold off on doing any cleanup on that side yet.

Reply:
Sounds good, I'll make those changes for this page and create an issue for the headers for the rest of Admin.

Reply:
- Headers in the component changed sizes to match recommendation
- Created #3064 to change over the rest of the admin headings
- Replaced caption size text with regular size